airbyte-integrations/connectors/source-mysql-v2/src/test/kotlin/io/airbyte/integrations/source/mysql/MysqlSourceDatatypeIntegrationTest.kt

```diff
@@ -168,9 +168,86 @@ class MysqlSourceDatatypeIntegrationTest {
                 "'OXBEEF'" to """"OXBEEF"""",
             )
 
+        val decimalValues =
+            mapOf(
+                "0.2" to """0.2""",
+            )
+
+        val intValues =
+            mapOf(
+                "10" to "10",
+                "100000000" to "100000000",
+                "200000000" to "200000000",
+            )
+
+        val dateValues =
+            mapOf(
+                "'2022-01-01'" to """"2022-01-01"""",
+            )
+
+        val timeValues =
+            mapOf(
+                "'14:30:00'" to """"14:30:00.000000"""",
+            )
+
+        val dateTimeValues =
+            mapOf(
+                "'2024-09-13 14:30:00'" to """"2024-09-13T14:30:00.000000"""",
+                "'2024-09-13T14:40:00+00:00'" to """"2024-09-13T14:40:00.000000""""
+            )
+
+        val timestampValues =
+            mapOf(
+                "'2024-09-12 14:30:00'" to """"2024-09-12T14:30:00.000000Z"""",
+                "CONVERT_TZ('2024-09-12 14:30:00', 'America/Los_Angeles', 'UTC')" to
+                    """"2024-09-12T21:30:00.000000Z"""",
+            )
+
+        val booleanValues =
+            mapOf(
+                "TRUE" to "true",
+                "FALSE" to "false",
+            )
+
+        // Encoded into base64
+        val binaryValues =
+            mapOf(
+                "X'89504E470D0A1A0A0000000D49484452'" to
+                    """"iVBORw0KGgoAAAANSUhEUgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=="""",
+            )
```

Comment:
I see a lot more test cases in the original source-mysql, why aren't you porting those over?

Reply:
Good call. I'll port them over.